### PR TITLE
SNS - Brake Pressure 

### PR DESCRIPTION
**Pull request by ishmis**
Added brake pressure tasks to observer + changed `can_listener.cpp` so c++20 methods are avoided

### CODE REVIEW

File: src/debugging/observer.cpp [hunk 1]
```diff
@@ -138,6 +139,24 @@ std::optional<std::unique_ptr<Observer>> Observer::fromFile(const std::string &p
...
+    if (!pins) {
+      log.error("failed to read brake pressure pin from file");
+      return std::nullopt;
+    }
+    for (const auto pin : *pins) {
+      observer->addBrakePressureTask(pin);
+    }
+  }
+
+
```
Comment (MiltFra) [inline src/debugging/observer.cpp:159]: too much whitespace

File: src/debugging/observer.cpp [hunk 2]
```diff
@@ -285,4 +304,36 @@ void Observer::addFakeTemperatureTask(const bool is_fail)
...
+void Observer::addBrakePressureTask(const uint8_t pin)
+{
+  auto brake_pressure = std::make_shared<sensors::BrakePressure>(pin);
+  std::stringstream name;
+  name << "brake_pressure-" << static_cast<uint32_t>(pin);
+  Task brake_pressure_task;
+  brake_pressure_task.name    = name.str();
+  brake_pressure_task.handler = [brake_pressure](JsonWriter &json_writer) {
+     brake_pressure->run();
+    json_writer.Key("value");
```
Comment (MiltFra) [inline src/debugging/observer.cpp:317]: indentation is off

**Review by mifrandir — CHANGES_REQUESTED** [review]
only small things

**Review by kshxtij — APPROVED** [review]

**Review by maxguy2001 — APPROVED** [review]
all good!

File: src/debugging/observer.cpp (hunk 2 above)
Comment (MiltFra) [reply src/debugging/observer.cpp:314]: bump

**Review by mifrandir — APPROVED** [review]

**Pull request merged**